lib/puppet_x/templates/init_ps.ps1.erb

```diff
@@ -286,8 +286,8 @@ function New-XmlResult
   $xml = [xml]@"
 <ReturnResult>
   <Property Name='exitcode'>$($exitcode)</Property>
-  <Property Name='errormessage'>$([System.Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes([string]$errormessage)))</Property>
```

Comment:
Already went down this road. You're going to want to keep the fully qualified namespace to support PowerShell v2.